packages/gestalt/src/TableOfContents/TableOfContentsAnchor.css

```diff
@@ -0,0 +1,35 @@
+.itemPadding {
+  padding-top: 13.5px;
```

Comment:
@hectoid can we make a change here and use integer value pixels? @rlingineni mentioned that it is dicouraged to use .5 pixels.